ansible/roles/operator-pipeline/templates/openshift/pipelines/community-signing-pipeline.yml

```diff
@@ -40,6 +40,7 @@ spec:
     - name: verify-whitelisted-index-image
       taskref:
         name: verify-whitelisted-index-image
+        bundle: "quay.io/redhat-isv/tkn-signing-bundle:3638365517"
```

Comment:
There is already a newer version of the bundle `3828559997`. Please update it here and below as well.